script.js

```diff
@@ -507,11 +507,29 @@ const form = document.getElementById('form');
 const email = document.getElementById('email');
 const error = document.getElementById('error');
 
+// form checking for submition
+
 form.addEventListener('submit', (e) => {
   if (email.value === email.value.toLowerCase()) {
     form.submit();
   } else {
     e.preventDefault();
     error.style.display = 'block';
   }
-});
\ No newline at end of file
+});
+
+// email validation
+
+const formElements = document.querySelectorAll('.storage');
+const stringValues = localStorage.getItem('data');
+const parseValues = JSON.parse(stringValues);
+
+[formElements[0].value, formElements[1].value, formElements[2].value] = parseValues;
+
+for (let i = 0; i < formElements.length; i += 1) {
+  formElements[i].addEventListener('keyup', () => {
+    const values = [formElements[0].value, formElements[1].value, formElements[2].value];
+    const stringifyValues = JSON.stringify(values);
+    localStorage.setItem('data', stringifyValues);
```

Comment:
- [ ] As the previous reviewer mentioned the data from the form doesn't persist in the local storage after the user inputs it, so when you refresh the page all the data is removed, kindly make sure to fix this so when the user inputs the data into the form it will also be saved to the local storage